src/common/_response-codes.yml

```diff
@@ -0,0 +1,18 @@
+description: |
+  The InfluxDB API uses standard HTTP status codes for success and failure responses.
+  The response body may include additional details. For details about a specific operation's response, see **Responses** and **Response Samples** for that operation.
+
+  API operations may return the following HTTP status codes:
+
+  | &nbsp;Code&nbsp; | Status                   | Description           |
```

Comment:
Why the non-breaking spaces here?

Reply:
To maintain the column width. When I shortened heading from "HTTP status" to "Code", I couldn't keep the cells below it from wrapping their text.